cargo-pgrx/src/templates/cargo_toml

```diff
@@ -4,7 +4,7 @@ version = "0.0.0"
 edition = "2021"
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "lib"]
```

Comment:
Why does everything need to also be a "lib" now?

Reply:
It's needed by `cargo run --bin schema` and I do not know how to force the lib to be compiled to crate-type lib in this command

Reply:
Hmm. I will fuss with that for a bit, later, then, since it should still be approximately as easy but require a slightly more obscure command (or maybe just executing the binary. Either way).